ow_lander/urdf/lander.xacro

```diff
@@ -544,12 +544,12 @@
     </visual>
     <selfCollide>1</selfCollide>
     <collision>
+      <max_contacts>1</max_contacts>
       <surface>
         <contact>
           <collide_bitmask>0x0001</collide_bitmask> <!-- Enable collision with the terrain by matching the bitmask -->
           <ode>
-            <kp>1e+15</kp>
-            <kd>1e+13</kd>
+            <kd>5e11</kd>

```

Comment:
removing kp just reverts it to the default value. Out of curiosity, what is the default value?

Reply:
default kp=1e+12 , kd=1 . More info here: http://sdformat.org/spec?ver=1.6&elem=collision#contact_ode